database.py

```diff
@@ -83,17 +93,49 @@ def is_not_phone_exists(phone):
     return not any(resp)
 
 
-# For example
-# create_bd()
-# insert_into_table('+34567890', 'Abbu', 'url1')
-# insert_into_table('+345678901', 'Abbu1', 'url2')
-# insert_into_table('+345678902', 'Abbu2', 'url3')
-# insert_into_table('+3456789025', 'Abbu2', 'https://e.mail.ru/messages/folder/5/')
-# insert_into_table('+3456789026', 'Abbu2', 'https://e.mail.ru/messages/folder/6/')
-# insert_into_table('+345678902266', 'Abbu2', 'https://e.mail.ru/messages/folder/6/', \
-                  # '30 800 &', 'https://e.mail.ru/messages', 'bla bla')
-
-# print filter_by_links(['1', 'url1', '2', 'url3', 'https://e.mail.ru/messages/folder/5/'])
-# print is_not_phone_exists('+345678901')
-# print is_not_phone_exists('+3456789')
-# print get_all()
+# Telegram
+def create_bd_telegram():
+    sql = """
+    CREATE TABLE if not exists telegram(
+        Id_external INTEGER,
+        Number VARCHAR(20) NOT NULL,
+        Name VARCHAR(100) NOT NULL,
+        Seen VARCHAR(30),
+        Profile VARCHAR(255),
+        CONSTRAINT unique_local UNIQUE (Number)
+        );
+    """
+    _exe_raw_sql(sql)
+
+
+# ToDo Drop hardcore
+def insert_into_telegram(number, name, _id=1, seen=None, profile=None):
```

Comment:
Maybe you could reuse existing method for insertion?

Reply:
Nope, we have the parameters logic here.
It's separate on both tables.